lib/pages/faq/view/faq_page.dart

```diff
@@ -158,6 +159,7 @@ class _QuestionsListState extends State<QuestionsList> {
                 padding: const EdgeInsets.fromLTRB(15, 0, 15, 15),
                 child: MarkdownBody(
                   fitContent: false,
+                  onTapLink: launch,
```

Comment:
Don't use `launch` directly, use `Utils.launchUrl`. It displays a message when something goes wrong.